src/python/examples/DisableUnit

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# vim:sw=4:ts=4:et
+from hirte import Hirte
```

Comment:
@dougsland 
This is not correct. 
Hirte defined in __init__ under src/python

You should call it like that,

>>> from src.python import Hirte
>>> hirte = Hirte()
>>> hirte.Disable(
...     "my-node-name",
...     ["chronyd.service", "hirte-agent.service"]
... )
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/yarboa/tests/hirte/src/python/__init__.py", line 107, in Disable


consider renaming src/python -> hirte-py

Reply:
> > 
> 
> Correct, That is what i got,
> 
> renamed: src/python/__init__.py -> src/python/hirte/__init__.py renamed: src/python/config.py -> src/python/hirte/config.py
> 
> If you use in setup url, setup.py should be under root directory or linked to your setup Applied this change in setup.py
> 
> - package_dir={"": "."}, - packages=find_packages("."), + package_dir={"": "src/python"}, + packages=find_packages(include=['hirte', 'hirte.src.python']),
> 
> And then pip install, will output the following /home/<user>/.local/lib/python3.11/site-packages/hirte-1.0.0.dist-info/
> 
> verify with command:
> 
> ```shell
> [yarboa@my-fedora hirte]$ python 
> Python 3.11.3 (main, May 24 2023, 00:00:00) [GCC 12.3.1 20230508 (Red Hat 12.3.1-1)] on linux
> Type "help", "copyright", "credits" or "license" for more information.
> >>> from hirte import Hirte
> ```

Thanks @Yarboa @pypingou